jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/dao/RDSDBManager.java

```diff
@@ -128,10 +123,11 @@ public static void insertNewPolicy(List<PolicyTable> policyList) {
                 "policyType=VALUES(policyType), " +
                 "severity=VALUES(severity), " +
                 "category=VALUES(category), " +
-                "status=VALUES(status), " +
                 "policyFrequency=VALUES(policyFrequency), " +
                 "resolutionUrl=VALUES(resolutionUrl)";
-
+        if (!excludeStatusUpdate.contains(datasource.toLowerCase())) {
```

Comment:
Why no simply check for null here? If status == null then ignore?

Reply:
Modified approach as per latest sync policies design